File: Exercism Codes/Python/protein_translation.py
def proteins(strand):
    lst = []
    for x in range(0, len(strand), 3):
        s = strand[x:x + 3]
        if s in ["AUG"]:
            lst.append("Methionine")
        elif s in ["UUU", "UUC"]:
            lst.append("Phenylalanine")
        elif s in ["UUA", "UUG"]:
            lst.append("Leucine")
        elif s in ["UCU", "UCC", "UCA", "UCG"]:
            lst.append("Serine")
        elif s in ["UAU", "UAC"]:
            lst.append("Tyrosine")
        elif s in ["UGU", "UGC"]:
            lst.append("Cysteine")
        elif s in ["UGG"]:
            lst.append("Tryptophan")
        elif s in ["UAA", "UAG", "UGA"]:
            break

    return lst

File: Exercism Codes/C#/ProteinTranslation.cs
using System;
using System.Collections.Generic;

public static class ProteinTranslation
{
    public static string[] Proteins(string strand)
    {
        List<string> lst = new List<string>();

        for(int i = 0; i < strand.Length; i += 3)
        {
            string s = strand.Substring(i, 3);
            switch(s)
            {
                case "AUG":
                    lst.Add("Methionine");
                    break;
                case "UUU":
                case "UUC":
                    lst.Add("Phenylalanine");
                    break;
                case "UUA":
                case "UUG":
                    lst.Add("Leucine");
                    break;
                case "UCU":
                case "UCC":
                case "UCA":
                case "UCG":
                    lst.Add("Serine");
                    break;
                case "UAU":
                case "UAC":
                    lst.Add("Tyrosine");
                    break;
                case "UGU":
                case "UGC":
                    lst.Add("Cysteine");
                    break;
                case "UGG":
                    lst.Add("Tryptophan");
                    break;
                case "UAA":
                case "UAG":
                case "UGA":
                    return lst.ToArray();
            }
        }

        return lst.ToArray();
    }
}